Add SSL pins and trackVerifiedAutoFailover docs for failover domain, configurable network timeout (#63)
radarlabs/mintlify: geofencing/fraud.mdx

To support the `sharing` flag on Android, pass `fraud = true` to `RadarInitializeOptions` in `Radar.initialize()`.

> To enable automatic failover to a secondary verified host if the primary host is erroring, also pass `trackVerifiedAutoFailover = true`:

Reviewer: ```suggestion
To enable automatic failover of `trackVerified()` requests to a secondary host if the primary host is unreachable, pass `trackVerifiedAutoFailover = true`:
```

If you do not install the plugin, `Radar.trackVerified()` will return `ERROR_PLUGIN`.

#### Initialize SDK

> To enable automatic failover to a secondary verified host if the primary host is erroring, also pass `trackVerifiedAutoFailover = true`, set `trackVerifiedAutoFailover` on `RadarInitializeOptions`:

Reviewer: ```suggestion
To enable automatic failover of `trackVerified()` requests to a secondary host if the primary host is unreachable, set `trackVerifiedAutoFailover = true` on `RadarInitializeOptions`:
```
